app/components/DeepNavBar.js

```diff
@@ -110,6 +116,9 @@ const baseStyles = _.extend(base.general, {
     statusBarPadding: {
         height: 16
```

Comment:
Might want to change this to 20. I used 20 height when I made yesterday's change.